constants/menuLinks.ts

```diff
@@ -190,7 +190,7 @@ const FOOTER_ELEMENTS: FooterStructure = [
     title: 'Support',
     elements: [
       SOCIAL_MEDIA.github,
-      SOCIAL_MEDIA.stack,
+      SOCIAL_MEDIA.support,
```

Comment:
I wonder whether we should mirror the items inside this column with the items on the `Help and Learning` column of the top menu (minus the Qiskit Textbook), namely `Support Channels` and `Qiskit Feedback`.
<img width="161" alt="Screenshot 2023-03-28 at 11 11 50" src="https://user-images.githubusercontent.com/7631333/228188403-1d2682ac-247f-4f2e-b611-0da4356fb278.png">

Reply:
To me, it makes sense... not sure who else's input we would need. 

CC: @eddybrando @JRussellHuffman ?